packages/govern-subgraph/schema.graphql

```diff
@@ -130,7 +130,7 @@ type Role @entity {
   frozen: Boolean!
 }
 
-enum PacketStatus {
+enum Status {
```

Comment:
Does it make sense to have `Vetoed` as a status? Should we instead set it to `Cancelled` [like in the contract](https://github.com/aragon/govern/blob/master/packages/govern-core/contracts/pipelines/GovernQueue.sol#L203) and, to store the different events, have a `Container.history = ContainerEvent[]` that would contain the entire history of events happening on a given container?

Reply:
Yes, that's a good point. I think we previously had a `Vetoed` state but I agree we should stick to the contracts. I like the idea of having a Container history 🙌